app/controllers/murals_controller.rb

```diff
@@ -20,6 +20,12 @@ def create
 
   def index
     @murals = Mural.all
+
+    if params[:search]
+      @murals = Mural.search(params[:search]).order("created_at DESC")
```

Comment:
This should work just fine, but you may want to use a SQL query here to get a better match for your user's search term!